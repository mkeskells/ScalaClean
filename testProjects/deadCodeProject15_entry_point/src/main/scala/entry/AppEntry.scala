package appentry

object Entry extends App {
  new Used()
}

class Used extends BaseUsed
class Unused extends BaseUnused

class BaseUsed
class BaseUnused

