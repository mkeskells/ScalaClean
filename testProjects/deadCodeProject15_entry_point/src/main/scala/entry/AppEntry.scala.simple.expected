package appentry

object Entry extends App {
  new Used()
}

class Used extends BaseUsed

class BaseUsed
class BaseUnused

